code/ui/blocks/src/blocks/Source.tsx

```diff
@@ -210,6 +253,12 @@ export const Source: FC<SourceProps> = (props) => {
     Please refer to the migration guide: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#source-block
   `);
   }
+  if (props.transformSource) {
+    deprecate(dedent`The \`transformSource\` prop on Source is deprecated and renamed, please use the \`transform\` prop instead. 
+    
+    Please refer to the migration guide: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#source-block
+  `);
+  }
```

Comment:
I thought we never had this prop in the first place?

Reply:
Ha, you're right. 🤦